mmv1/products/kms/CryptoKey.yaml

```diff
@@ -85,6 +85,34 @@ properties:
     name: 'labels'
     description: |
       Labels with user-defined metadata to apply to this resource.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'primary'
+    description: |
+      A copy of the primary CryptoKeyVersion that will be used by cryptoKeys.encrypt when this CryptoKey is given in EncryptRequest.name.
+      The CryptoKey's primary version can be updated via cryptoKeys.updatePrimaryVersion.
+      Keys with purpose ENCRYPT_DECRYPT may have a primary. For other keys, this field will be omitted.
+    output: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          The resource name for this CryptoKeyVersion.
+      - !ruby/object:Api::Type::Enum
```

Comment:
For output-only fields, we don't use the enum values or display them to users - it's generally better to use a string type for the field and link to the API docs for the enum directly (then they also don't get out of sync)